videointelligence/noxfile.py

```diff
@@ -118,19 +118,11 @@ def system(session):
     session.install("-e", "../test_utils/")
     session.install("-e", ".")
 
-    # Additional set up for VPC SC.
-    env = {
-        "PROJECT_NUMBER": "570941833855",
-        "GOOGLE_CLOUD_TESTS_VPCSC_OUTSIDE_IP": "10.1.1.1",
-        "GOOGLE_CLOUD_TESTS_VPCSC_INSIDE_IP": "55.55.0.0",
-    }
     # Run py.test against the system tests.
     if system_test_exists:
-        session.run("py.test", "--quiet", system_test_path, env=env, *session.posargs)
+        session.run("py.test", "--quiet", system_test_path, *session.posargs)
     if system_test_folder_exists:
-        session.run(
-            "py.test", "--quiet", system_test_folder_path, env=env * session.posargs
-        )
+        session.run("py.test", "--quiet", system_test_folder_path, *session.posargs)
```

Comment:
Reverts #8607.

Reply:
Ah oops, I thought we'd excluded this in the synth. Thank you!